src/System.Reflection.Metadata.Cil/src/Visitor/ICilVisitor.cs

```diff
@@ -2,6 +2,7 @@
 
 namespace System.Reflection.Metadata.Cil.Visitor
 {
+    [CLSCompliant(false)]
```

Comment:
Any reason you didn't must mark the entire assembly as CLSCompliant(false)? or disable the check?